step.yml

```diff
@@ -90,3 +90,13 @@ inputs:
         The short description of the status.
 
         If left empty, it will be the status of the build.
+  - coverage:
+    opts:
+      title: "Coverage"
+      summary: "Test coverage"
+      description: |-
+        The test coverage.
+
+        Must be a representation of a floating number. If not, it will be discarded. The minimum is 0.0 and the
```

Comment:
This is shorter:  "Must be a floating point number between 0.0 and 100.0."

Reply:
Updated.